src/esp/gfx_batch/Renderer.cpp

```diff
@@ -41,7 +42,9 @@
 #include <Magnum/Trade/PhongMaterialData.h>
 #include <Magnum/Trade/SceneData.h>
 #include <Magnum/Trade/TextureData.h>
+#include <esp/gfx_batch/DepthUnprojection.h>
 #include <unordered_map>
+#include "esp/core/Check.h"
```

Comment:
This library doesn't depend on `esp::core`, so you can't use this include here. See the CMakeLists for more info.